### PR TITLE
Added EAS fits + Kiki Keres/Hyena.

### PR DESCRIPTION
**Pull request by Epicloa-Program**
Fixed small description error on Crucifier. Merged ADV-5 Keres into the ADV-30 fit since they were 95% the same.

### CODE REVIEW

File: Fits/Electronic Attack Frigates/[ADV-30] Hyena.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+Advanced EWAR Frigate
+
+## Description
+
+Utilizes Target Painters to enhance the friendly fleet's DPS against a hostile target. This ship is entirely reliant on range to avoid incoming DPS so do not burn close unless absolutely necessary. Use 3x TP if purely fleet support, mix in webs if it's going to be possible to get close enough to apply them.
+
+## Fit
+```
+[Hyena, [ADV-30] Hyena]
```
Comment (bearthatcares) [inline Fits/Electronic Attack Frigates/[ADV-30] Hyena.md:11]: Hyena should definitely be shield and speed, I think?

File: Fits/Electronic Attack Frigates/[ADV-30] Keres.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+Small Ancillary Armor Repairer
+
+5MN Cold-Gas Enduring Microwarpdrive
+Remote Sensor Dampener II
+Remote Sensor Dampener II
+Remote Sensor Dampener II
+Phased Muon Scoped Sensor Dampener
+
+Prototype Cloaking Device I
+[Empty High slot]
```
Comment (bearthatcares) [inline Fits/Electronic Attack Frigates/[ADV-30] Keres.md:31]: Auto targeting might be good, gives more targets

File: Fits/Electronic Attack Frigates/[ADV-30] Kitsune.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+If you know what the enemy is going to be fielding on grid, use the list below to pick the appropriate ECM modules since they are stronger against their specific targets.
+
+Ladar - Minmatar, Angel Cartel.
+Radar - Amarr, Blood Raiders, Sansha's Nation, Triglavian Collective.
+Magnometric - Gallente, ORE, Sisters of Eve.
+Gravimetric - Caldari, Guristas, Mordu's Legion.
+
+## Fit
+```
+[Kitsune, [ADV-30] Kitsune]
```
Comment (bearthatcares) [inline Fits/Electronic Attack Frigates/[ADV-30] Kitsune.md:18]: I definitely think T2 ECM should be a requirement for the Kitsune, it's way better

Comment (Epicloa-Program) [reply Fits/Electronic Attack Frigates/[ADV-30] Kitsune.md:18]: > I definitely think T2 ECM should be a requirement for the Kitsune, it's way better

The primary issue was cost. it takes it from a 43.3mil fit to a 74.8mil fit with refits, and the actual % jam chance increase is really not there for a 2x price increase. Example on the Absolute Honor CFI (1sigamp) with mindlinked info bursts:

T2:

1 jam - 22%
2 jam - 39.2%
3 jam - 52.5%
4 jam - 63%

Scoped:

1 jam - 19.8%
2 jam - 35.7%
3 jam - 48.4%
4 jam - 58.6%

So it's basically a 2.2% - 4.4% increase in % chance from scoped to T2 depending on how many jams you have on them, which just didn't eel like it was worth nearly doubling the price for.

File: Fits/Electronic Attack Frigates/[ADV-30] Keres.md (hunk 2 above)
Comment (Epicloa-Program) [reply Fits/Electronic Attack Frigates/[ADV-30] Keres.md:31]: They get 9 targets by default so it felt overkill lol

File: Fits/Electronic Attack Frigates/[ADV-30] Hyena.md (hunk 1 above)
Comment (Epicloa-Program) [reply Fits/Electronic Attack Frigates/[ADV-30] Hyena.md:11]: Goal was really just to maximize the EWAR effectiveness of these and maximize range, this thing operates comfortably at like 130km so I think tank is kind of secondary. If it gets caught 1k EHP isn't going to save it realistically and you lose a lot of what it is actually on grid to do.

File: Fits/Electronic Attack Frigates/[ADV-30] Kitsune.md (hunk 3 above)
Comment (bearthatcares) [reply Fits/Electronic Attack Frigates/[ADV-30] Kitsune.md:18]: Huh? T2 ECM mods are like 2m each, this should be an 8m difference

Comment (Epicloa-Program) [reply Fits/Electronic Attack Frigates/[ADV-30] Kitsune.md:18]: > Huh? T2 ECM mods are like 2m each, this should be an 8m difference

You need the refits as well. It's 4x sets of 4 mods.

File: Fits/Electronic Attack Frigates/[ADV-30] Hyena.md (hunk 1 above)
Comment (Epicloa-Program) [reply Fits/Electronic Attack Frigates/[ADV-30] Hyena.md:11]: Fixed, kept the 130km range but got some more speed/agility.

**Comment by Epicloa-Program** [comment]
Fixing PRs to avoid conflicts.

**Pull request closed without merging**